### ISSUE TITLE
NVDA lag/poor responsiveness when sharing desktop on Skype for Business, Microsoft Teams

### ISSUE THREAD

**Issue by robinettl** [nvaccess/nvda#10178]
### Steps to reproduce:
When sharing my Windows desktop in a Skype for Business or Microsoft Teams call with coworkers, NVDA becomes slow and unresponsive.  I will press an NVDA keyboard shortcut and nothing will happen, or it will take several seconds for that command to execute.  Under normal circumstances, tapping the shift key will pause NVDA speech, but in these situations, NVDA either responds very late to the shift key or never responds.  This makes it difficult and sometimes impossible to collaborate using these conferencing programs, and unfortunately I don't have the option of using a different conferencing platform because these are what the company uses nationwide.

**Comment by seanbudd** [comment]
Hi @robinettl - is this still an issue for you? I've updated your issue to use our issue template format. Please fill it out if this is still an issue.

**Comment by Adriani90** [comment]
I cannot reproduce this issue with NVDA 2022.4 and Microsoft Teams 1.6 or with Skype for Business. I am closing this issue as abandoned, since we didn
t get any updates from the initial author since more than 3 years. Feel free to comment on this if it is still an issue for you and we can reopen.
